bot/cogs/snakes.py

```diff
@@ -29,6 +34,25 @@ def __init__(self, bot: AutoShardedBot):
         :return: A dict containing information on a snake
         """
 
+        # Getting the summary
```

Comment:
This function still needs to give a random snake when `name is None`, and also it needs to special-case Python and return information about the programming language.

Additionally, this method is case-sensitive.